src/jimgw/core/single_event/likelihood.py

```
@@ -813,64 +838,43 @@ def _max_phase_diff(
-        phase_diff_array = self._max_phase_diff(freqs, freqs[0], freqs[-1], chi=chi)
-        total_phase = phase_diff_array[-1]
-        phase_diff = jnp.linspace(0.0, total_phase, n_bins + 1)
-        f_bins = interp1d(phase_diff_array, freqs)(phase_diff)
-        f_bins_center = (f_bins[:-1] + f_bins[1:]) / 2
-        return jnp.array(f_bins), jnp.array(f_bins_center)
-
     @staticmethod
     def _compute_coefficients(
-        data: Complex[Array, " n_freq"],
+        detector: Detector,
         h_ref: Complex[Array, " n_freq"],
```

Ruff (0.15.20)

[error] Syntax error in forward annotation: Expected an expression

(F722)